nixos/tests/gitlab.nix

```diff
@@ -6,7 +6,7 @@
 # - Creating Merge Requests and merging them
 # - Opening and closing issues.
 # - Downloading repository archives as tar.gz and tar.bz2
-import ./make-test-python.nix ({ pkgs, lib, ... }:
+{ pkgs, lib, ... }:
```

Comment:
```suggestion
# Run with
# [nixpkgs]$ nix-build -A nixosTests.gitlab

{ pkgs, lib, ... }:
```

`nix-build <file>` is not possible anymore, so this helps both newcomers and old hands.
It's documented in the manual, but that's far away.